<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Categories and Terms</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QPushButton" name="EditTermButton">
      <property name="text">
       <string>Edit Terms</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QTableView" name="TermTableView">
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="EditCategoriesButton">
      <property name="text">
       <string>Edit Categories</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="TermCategoriesListWidget"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="TestButton">
      <property name="text">
       <string>Test On Selected Category</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionCategories_with_text_file"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGet_Help"/>
   </widget>
   <addaction name="menuAdd"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTerms_with_text_file">
   <property name="text">
    <string>Terms with text file</string>
   </property>
  </action>
  <action name="actionCategories_with_text_file">
   <property name="text">
    <string>Categories with text file</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionGet_Help">
   <property name="text">
    <string>Get Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
